packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -0,0 +1,178 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+
+import '../../all-components';
+import type { WaferMapDie, WaferMapColorsScale } from '../types';
+import {
+    WaferMapQuadrant,
+    WaferMapOrientation,
+    WaferMapColorsScaleMode
+} from '../types';
+import {
+    HIGHLIGHTEDVALUESETS,
+    WAFERMAPDIESETS,
+    WAFERMAPCOLORSCALESETS
+} from './sets';
+
+interface WaferMapArgs {
+    quadrant: WaferMapQuadrant;
+    orientation: WaferMapOrientation;
+    maxCharacters: number;
+    dieLabelsHidden: boolean;
+    dieLabelsSuffix: string;
+    colorsScaleMode: WaferMapColorsScaleMode;
+    highlightedValues: number[];
+    dies: WaferMapDie[];
+    colorScale: WaferMapColorsScale;
+}
+
+const metadata: Meta<WaferMapArgs> = {
+    title: 'Wafermap',
```

Comment:
There are other places in files with "WaferMap" with "m" instead of "M". Also in the lines 35, 44, 71

Reply:
applied